2.4/examples/replica/README.md

```diff
@@ -0,0 +1,64 @@
+# MongoDB Replica Set example
+
+**WARNING:**
+
+**This is only a Proof-Of-Concept example and it is not ment to be used in any
+production. Use at your own risk.**
+
+## What is MongoDB Replica Set?
+
+A cluster of MongoDB servers that implements master-slave replication and automated failover.
+MongoDB’s recommended replication strategy.
+
+## Build Docker image:
+
+* `git clone https://github.com/mfojtik/mongodb && cd mongodb`
+* `git checkout replica`
+* `cd ./2.4 && docker build -t openshift/mongodb-24-centos7:replica .`
+
+## Deployment:
+
+In order to run this Docker image, you need to have the latest version of
+OpenShift Origin v3.
+
+* `openshift start --latest-images &> server.log &`
+* `osc create -f ./2.4/examples/replica/mongo_controller.json`
+
+## How does it work?
+
+The MongoDB replication controller is set to **1** in this example. Once the Pod
+with MongoDB is started, the replica set is initialized (`rs.initiate()`)
+and the Pod is elected as the 'master' (as there are no other MongoDB Pods
+running, yet).
+
+If you want to add new node to MongoDB replica set, bump the value of 'replicas'
+in the Kubernetes Replication Contoller to '2'.
```

Comment:
i wonder if we can do better using the deployment hooks that dan mace is working on to just a hook in exactly one pod.